smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/StructureGenerator.kt

```diff
@@ -283,16 +233,8 @@ class StructureGenerator(
                     }
                 }
 
-                // generate the Java builder overrides
-                // NOTE: The enum overloads are the same in both the Java and DslBuilder interfaces, generating
-                // the Java builder implementation will satisfy the DslInterface w.r.t enum overloads
                 write("")
-                write("override fun build(): #class.name:L = #class.name:L(this)")
-                for (member in sortedMembers) {
-                    val (memberName, memberSymbol) = memberNameSymbolIndex[member]!!
-                    // we want the type names sans nullability (?) for arguments
-                    write("override fun #1L(#1L: #2T): FluentBuilder = apply { this.#1L = #1L }", memberName, memberSymbol)
-                }
+                write("fun build(): #class.name:L = #class.name:L(this)")
```

Comment:
same question, should this be `internal`?

Reply:
In testing I didn't see an issue with making this internal, will update.